make/run

```diff
@@ -32,10 +32,19 @@ kubectl get storageclass persistent 2>/dev/null || {
     kubectl create -f -
 }
 
-kubectl create --namespace="${NAMESPACE}" \
-    --filename="kube/bosh" \
-    --filename="kube/secrets" \
-    --filename="kube/bosh-task/post-deployment-setup.yml"
+source bin/settings/settings.env
+source bin/settings/network.env
+namespace=${NAMESPACE}
+source bin/settings/kube/hcp-compatibility.env
```

Comment:
Do we still have things in there that are useful? Huh. We should consider moving them to the main settings env (possibly in a different PR)